tests/mocks/valid_docker_stack.yaml

```diff
@@ -2,18 +2,19 @@ version: "3"
 
 services:
   app:
-    image: jenkins:latest
+    image: alpine:latest
```

Comment:
minor: As a general rule, I would avoid using `latest` tag in here. I think you do not want to see if the test breaks if the ubuntu version changed and has a bug. I would pin it to something that does not get overwritten continuously.

Reply:
good point thx